depends/packages/freetype.mk

```diff
@@ -1,8 +1,8 @@
 package=freetype
-$(package)_version=2.7.1
+$(package)_version=2.10.4
```

Comment:
Why not 2.11.1?

Reply:
> Why not 2.11.1?

Because that version of freetype doesn't exist? Have moved this to 2.11.0 though.